content/en/docs/tasks/configure-pod-container/configure-pod-initialization.md

```diff
@@ -80,6 +80,11 @@ The output shows that nginx is serving the web page that was written by the init
       ...
 
 
+Exit the shell.
+
+Delete the Pod that you created for this exercise:
```

Comment:
```suggestion
Delete the Pod that you created for this exercise (optional):
```
It might be helpful to make this step optional if the platform being used is cleared up on its own (ex. Katacoda/Play With Kubernetes platforms)